content/acra/configuring-maintaining/general-configuration/acra-addzone.md

```diff
@@ -60,6 +60,18 @@ weight: 10
 
   Password to Redis database.
 
+### Keystore
+
+* `--keystore_encryption_type=<strategy>`
+
+  Keystore encryption strategy.
+  Currently supported strategies:
+  * **`master_key`** (**Default**) - Keystore using Acra Master Key, loaded from ENV (`ACRA_MASTER_KEY`) variable;
+  * **`vault_master_key`** -  Keystore using Acra Master Key, loaded from Hashicorp Vault
+  * **`kms_encrypted_master_key`** - Keystore using Acra Master Key, loaded from ENV `ACRA_MASTER_KEY` variable and decrypted
+    via KMS key-encryption key.
+  * **`kms_per_client`** - Keystore using KMS for decryption Acra keys per ClientID and ZoneID.
+    Create new KMS zone key-encryption key if not present on KMS.
```

Comment:
```suggestion
  * **`env_master_key`** (**Default**) - Keystore using Acra Master Key, loaded from ENV (`ACRA_MASTER_KEY`) variable.
  * **`vault_master_key`** -  Keystore using Acra Master Key, loaded from Hashicorp Vault.
  * **`kms_encrypted_master_key`** - Keystore using Acra Master Key, loaded from ENV `ACRA_MASTER_KEY` variable and decrypted via KMS key-encryption key.
  * **`kms_per_client`** - Keystore using KMS for decryption Acra keys per ClientID and ZoneID.
    Create new KMS zone key-encryption key if not present on KMS.
```

i suggest to rename `master_key` to `env_master_key`?

Reply:
So we need to change it in Acra too.

Reply:
@Lagovas take a look, wdyt?

Reply:
yes, looks better, cool suggestion. lets do it in acra too, because looking on kms_*, vault_* it ask us to use prefixes)